<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="LabelSettings">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Настроки</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>440</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>440</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="GroupSettings">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>611</width>
     <height>251</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="GeneralSettings">
    <attribute name="title">
     <string>Общие</string>
    </attribute>
    <widget class="QPushButton" name="ChangePGDumpPathButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>21</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="DBMaintenceLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Пусть к pg_dump</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PGDumpPath">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>371</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="DBPollingFreqSlider">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>15</number>
     </property>
     <property name="maximum">
      <number>55</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksAbove</enum>
     </property>
    </widget>
    <widget class="QTimeEdit" name="DBMaintenceTimeEdit">
     <property name="geometry">
      <rect>
       <x>197</x>
       <y>90</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="DBPollingFreqSliderLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Частота опроса БД</string>
     </property>
    </widget>
    <widget class="QLabel" name="DBPollingFreqSliderLabel_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Обслуживание БД</string>
     </property>
    </widget>
    <widget class="QLabel" name="GetSliderLabel">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="SendingSettings">
    <attribute name="title">
     <string>Оповещение</string>
    </attribute>
    <widget class="QGroupBox" name="MailGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>581</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Тут будет почта</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="TelegramGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>581</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Тут будет телегабот</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
